spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -402,12 +403,12 @@ def update_request
 
     context "with a valid replace put operation" do
       before(:each) { update_request }
-      let(:new_display_name) { "Mr_Creosote" }
+      let(:new_login) { "Mr_Creosote" }
```

Comment:
Are we going to allow people to change their login names? If this is going to be used as the URL slug as per [here](https://github.com/zooniverse/Panoptes-Front-End/issues/513#issuecomment-110486503) then I don't think we should let the login field be updated.